packages/cloudflare/README.md

```diff
@@ -186,17 +186,21 @@ export default defineConfig({
 
 ### `runtime`
 
-`runtime: "off" | "local"`
+`runtime: { mode: "off" | "local", persistTo: string }`
 
-default `"off"`
+default `{ mode: 'off', persistTo: '' }`
 
 Determines whether and how the Cloudflare Runtime is added to `astro dev`.
 
 The Cloudflare Runtime includes [Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/bindings), [environment variables](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables), and the [cf object](https://developers.cloudflare.com/workers/runtime-apis/request/#incomingrequestcfproperties). Read more about [accessing the Cloudflare Runtime](#cloudflare-runtime).
 
+The `mode` property defines how the runtime is added to `astro dev`:
+
 - `local`: uses bindings mocking and locally static placeholders
 - `off`: no access to the Cloudflare runtime using `astro dev`. You can alternatively use [Preview with Wrangler](#preview-with-wrangler)
 
+The `persistTo` property defines where the local runtime is persisted to. This is only relevant when using `mode: local`. The default value is `.wrangler/state/v3` which means the files are safed to the directory `.wrangler/state/v3` relative to your astro dev execution path. This is the default path wrangler uses, so both tools should be able to pickup and work with the local state. You can change this to any other directory, however this directory should be included in the `.gitignore`.
```

Comment:
```suggestion
The `persistTo` property defines where the local runtime is persisted to when using `mode: local`. This value is a directory relative to your `astro dev` execution path.

The default value is set to `.wrangler/state/v3` to match the default path Wrangler uses. This means both tools are able to access and use the local state. 

If you change this value to any other directory, be sure to include it in `.gitignore`.
```
Took a shot from a straight language perspective, since I'm not familiar with the process itself. Make these sentences correct, but my thoughts are:

- that was a big, dense paragraph. Splitting up into what it is, the info about the default, and configuring something other than the default

- unless these are precise or common technical terms, "pick up" and "work with" are slightly colloquial English phrase that will be less easy to interpret or translate by non-native speakers. Precise terms, even if they sound drier or "more difficult", are often more accessible because they will be more standard and easier to look up if necessary. I inferred this meant something like "access and use" but choose whatever precisely is happening here. "detect"? "find"? whatever is correct!

- You would hate the way to properly use "however" in a sentence, so I rewrote to avoid.  :joy: To use properly it is written as:

> ... any other directory; however, this directory ...

So, adjust to taste, but I think by virtue of saying "if you do" implies that you CAN do it, so maybe you don't need to both say that it's possible, and then what you have to do if you choose to do it.

Reply:
Totally agree, the only thing I changed is, that the user even has to put the default directory to `.gitignore` themself.